pkg/utils/release/secrets.go

```diff
@@ -0,0 +1,29 @@
+package release
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Contains all methods related with secret objects CRUD operations.
+type SecretsInterface interface {
+	//Creates a registry secret.
+	CreateRegistryJsonSecret(name, namespace, authKey, keyName string) (*corev1.Secret, error)
+}
+
+// CreateRegistryJsonSecret creates a secret for registry repository in namespace given with key passed.
+func (r *releaseFactory) CreateRegistryJsonSecret(name, namespace, authKey, keyName string) (*corev1.Secret, error) {
```

Comment:
this kind of functions should live in utils/common/secrets

Reply:
Moved.